swim/disseminator.go

```diff
@@ -137,7 +137,7 @@ func (d *disseminator) bumpPiggybackCounters(changes []Change) {
 			continue
 		}
 
-		c.p += 1
+		c.p++
```

Comment:
is this a lint thing? I like the other one better. But don't mind having it like this either